docs/one-click-integrations/jira-integration.md

```diff
@@ -1,29 +1,96 @@
 # Jira integration
 
-Pulse integrates with Jira Cloud to receive data about issues, necessary to calculate the metrics [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+Pulse integrates with Jira Cloud to receive data about:
+
+-   **Issues** - necessary to calculate [Lead time and Cycle time](../metrics/lead-cycle-time.md).
+
+-   **Incidents** - necessary to calculate the following Accelerate metrics:
```

Comment:
The actual objects that the integration retrieves from JIra are just issues (meaning that the "incidents" are actually also issues tagged with a special label, not a different object in Jira). So a "truer" way of saying this would be like this:

```suggestion
Pulse integrates with Jira Cloud to receive data about **issues**, necessary to calculate [Lead time and Cycle time](../metrics/lead-cycle-time.md) and the following Accelerate metrics:
```

Reply:
@paulo Thank you for the suggestion. 🙏  However, "incident" is the type of event that Pulse retrieves from Jira issues to calculate Time to recover and Change failure rate metrics. The documentation for the other integrations, GitHub and Bitbucket, follows the same approach. Thus, as we talked, I'll keep it as it is.